### ISSUE TITLE
.NET Core master branches have switched to "5.0"

### ISSUE THREAD

**Issue by richlander** [dotnet/announcements#118]
# .NET Core master branches have switched to "5.0"

Various .NET Core repos have switched their `master` branches to "5.0", and will use 3.x branches for 3.0 and 3.1 projects. This is very similar to how we approached 2.x and 3.0 projects. As context, `master` branches produced 3.0 builds for the entire period of the 2.2 project.

At present, this change is only relevant if you contribute to .NET Core repositories. Otherwise, we recommend that you wait for the 5.0 Preview releases before installing this new version.

## Discussion
Please share your thoughts on this change and see what others are saying at:

[dotnet/core #3059](https://github.com/dotnet/core/issues/3059)

## Details

The following repos have moved their master branches to "5.0":

- [dotnet/coreclr](https://github.com/dotnet/coreclr)
- [dotnet/corefx](https://github.com/dotnet/corefx)
- [dotnet/core-setup](https://github.com/dotnet/core-setup)

You can see that [5.0 .NET Core Runtime builds](https://github.com/dotnet/core-setup#daily-builds) are now being built and published at dotnet/core-setup.

In large part, we switched `master` to 5.0 to enable to the community to contribute changes. As the .NET Core 3.0 released has progressed, we've had to increase scrutiny on the changes we will accept. Now that we've permanently branched for 3.0, we can accept a much broader set of changes. Please consider that to be an open invitation!

Notably, [dotnet/core-sdk](https://github.com/dotnet/core-sdk) is not yet producing 5.0 builds. We expect that the dotnet/core-sdk master branch will continue as a 3.0 branch for 2-3 months while we finish up the .NET Core 3.0 project.

At present, there is no developer experience for 5.0. We are waiting on the dotnet/core-sdk repo to switch to 5.0. Switching your project file to target `netcoreapp5.0` won't do anything useful. We'll update this issue when a 5.0 SDK comes available.
